tests/general/assert-fail.c.stderr.expected

```diff
@@ -0,0 +1,7 @@
+Assertion failed: false (assert-fail.c: main: 5)
+Error: failed to run main module `assert-fail.c.---.wasm`
+
+Caused by:
+    0: failed to invoke `_start`
+    1: wasm trap: unreachable, source location: @----
+       wasm backtrace:
```

Comment:
Where is this output coming from?  Is it coming from some wasmtime-internal place?

Reply:
Yeah. If other engines have a different format, we can adjust the filter script as needed.